ui/app/templates/components/kv-object-editor.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="field" ...attributes>
+<div class="field" ...attributes {{did-insert this.createKvData @value}}>
```

Comment:
This came up quite a few times where accessing a property twice in the same computation was being thrown. I used this pattern and moved the logic from the constructor to an action rather than leaving it and using `next`.